### ISSUE TITLE
Test FiniteVolumeMethod is not working. Your examples are not running.

### ISSUE THREAD

**Issue by MrBenzWorld** [DanielVandH/FiniteVolumeMethod.jl#21]

**Comment by DanielVandH** [comment]
What's the error that you get? The examples require Gmsh for meshing, so you need to also make sure a Gmsh path is correctly set - the code currently uses

```
GMSH_PATH = "./gmsh-4.9.4-Windows64/gmsh.exe"
```

**Comment by MrBenzWorld** [comment]
yes i have corrected the path for GMSH_PATH .

this is the error.
LoadError: UndefVarError: getx not defined

 
Thank you for the response.

**Comment by DanielVandH** [comment]
Ah, yikes - I think it's because I forgot to list DelaunayTriangulation in the list of packages before the examples. Can you do `using DelaunayTriangulation` and see if that works?

**Comment by MrBenzWorld** [comment]
yes, I corrected it. now working. 

Could you please add one example or help to do it?

oscillating flow through pipe (2D)
it is a transient compressible, viscous 2D flow problem.  also, conventional heat transfer considers from the pipe wall. 
it is a convention diffusion problem.  

I am new to Julia. I want to do this with Julia only.
Thank for provig FiniteVolumeMethod.  I was trying but did not get proper help from other packages in Julia.

Thank you.
@DanielVandH

**Comment by DanielVandH** [comment]
Great! Thanks for the issue. I'll update the README soon.

I'm happy to try and provide an example for you - can you provide the complete PDE and boundary conditions, along with the geometry you have in mind? Like e.g. what I do here

![image](https://user-images.githubusercontent.com/95613936/208846376-493636a2-58b8-48da-b129-474024ad567f.png)

(or in any of the other examples).

**Comment by MrBenzWorld** [comment]
![equation](https://user-images.githubusercontent.com/113277019/208850287-348d77a6-748e-4615-9e63-be9542bd7484.jpg)
![PT1](https://user-images.githubusercontent.com/113277019/208850705-8d6aab54-710d-43a4-b4d0-f2907d416845.jpg)

**Comment by MrBenzWorld** [comment]
at both ends, there are porous heat exchangers. 
Thank you very much for your help.

if possible, could you provide a contact email?

**Comment by DanielVandH** [comment]
Unfortunately, it's not possible to implement that currently with my code - there's no support for systems, only scalar problems. Sorry about that.

Maybe there's another package that could help you here https://github.com/JuliaPDE/SurveyofPDEPackages? I think VoronoiFVM.jl supports systems, though I've never used it.

**Comment by MrBenzWorld** [comment]
is it possible to only pulse tube without heat exchangers with porosity?
 I have tried VoronoiFVM but did not succeed. 
Thank you

**Comment by DanielVandH** [comment]
I'm not sure what you mean sorry, can you rephrase or show the reduced equation you have in mind?

**Comment by MrBenzWorld** [comment]
Here, I can add one-dimensional equations ,
Please check, 
![4](https://user-images.githubusercontent.com/113277019/208861280-25351a9e-13b3-4bd8-adfc-be6a3a9b3925.jpg)
Thank you.

**Comment by DanielVandH** [comment]
Sorry, by scalar I still mean two-dimensional, just can't have coupled equations. Don't think I yet have the tools you need unfortunately. Maybe ask on the Julia Discourse with your PDEs and see if anyone has any pointers on what other tools might be available for you? They're usually pretty helpful over there.

Hope you can find what you need!

**Comment by MrBenzWorld** [comment]
Thank you very much.

Please add examples in the example folder.
Please don't keep all the examples on the webpage. it is taking a lot of time to load. also, crashing chrome.

**Comment by DanielVandH** [comment]
Will do - my plan is to put it all into a proper docs soon.

**Comment by DanielVandH** [comment]
Done now https://danielvandh.github.io/FiniteVolumeMethod.jl/dev/. Hopefully there's no more performance issues now - thanks for letting me know that that was happening @MrBenzWorld.
